pkgs/development/ocaml-modules/irmin/chunk.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, buildDunePackage, io-page-unix, irmin, irmin-mem, irmin-test }:
+
+buildDunePackage rec {
+
+  pname = "irmin-chunk";
+  inherit (irmin) version src useDune2;
+
+  propagatedBuildInputs = [ io-page-unix irmin ];
```

Comment:
Is `io-page-unix` really needed?

Reply:
No? I can build it without, I'm not sure how it got in there.